libobs/obs-encoder.c

```diff
@@ -710,6 +837,30 @@ void obs_encoder_set_scaled_size(obs_encoder_t *encoder, uint32_t width,
 	encoder->scaled_height = height;
 }
 
+void obs_encoder_enable_gpu_scaling(obs_encoder_t *encoder, bool enable,
+				    enum obs_scale_type scale_type)
+{
+	if (!obs_encoder_valid(encoder, "obs_encoder_enable_gpu_scaling"))
+		return;
+	if (encoder->info.type != OBS_ENCODER_VIDEO) {
+		blog(LOG_WARNING,
+		     "obs_encoder_enable_gpu_scaling: "
+		     "encoder '%s' is not a video encoder",
+		     obs_encoder_get_name(encoder));
+		return;
+	}
+	if (encoder_active(encoder)) {
+		blog(LOG_WARNING,
+		     "encoder '%s': Cannot enable GPU sclaing "
```

Comment:
```suggestion
		     "encoder '%s': Cannot enable GPU scaling "
```

Reply:
Fixed in https://github.com/obsproject/obs-studio/pull/9043/commits/373a24f4b8bc29d6519e74a38041d6e6b67cac70